input/k8s/nodes.go

```diff
@@ -140,14 +140,21 @@ func publishK8sNodes(ctx context.Context, log *logp.Logger, publisher stateless.
 		o, ok := obj.(*kube.Node)
 		if ok {
 			log.Debugf("Publish Node: %+v", o.Name)
+			conditions := o.Status.Conditions
+			statusReady := "false"
+			for _, condition := range conditions {
+				if condition.Type == "Ready" {
+					statusReady = "true"
+				}
```

Comment:
Shouldn't we also look if `condition.Status=true` here?

Reply:
You are completely right. Thanks for spotting this !